File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/VM_USER_STATUS.ts
export class VM_USER_STATUS {
    public USER_ID: string;
    public FIRST_NAME: string;
    public LAST_NAME: string;
    public ACCOUNT_DISABLED: any;
    public PROFILE_ID: string;
    public ORG_GROUP_ID: string;
    public CREATE_DATE: string;
    public EMAIL_ID: string;
    public PASSHASH_REQUIRED: boolean;
    public TOKEN_EXPIRY_PERIOD: number;
    public IDLE_TIME: number;
    public MIDDLE_INITIAL: string;
    public PHONE1: string;
    public PHONE2: string;
    public FAX: string;
    public LDAP_USER: string;
    public LDAP_ROLE: string;
    public LDAP_ORG: string;
    public REPORT_USER: boolean;   
    public USERDN: string;   
    public PAGER: string; 
    public isDisabled: boolean;
    public isHidden: boolean;
}

File: RSPAClient/Core/AtPar.ViewModel/VM_USER_STATUS.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AtPar.ViewModel
{
    public class VM_USER_STATUS
    {
        public string USER_ID { get; set; }
        public string FIRST_NAME { get; set; }
        public string LAST_NAME { get; set; }
        public Nullable<System.DateTime> CREATE_DATE { get; set; }
        public bool ACCOUNT_DISABLED { get; set; }
        public string PROFILE_ID { get; set; }
        public string ORG_GROUP_ID { get; set; }
        public string EMAIL_ID { get; set; }
        public bool PASSHASH_REQUIRED { get; set; }
        public Int32 TOKEN_EXPIRY_PERIOD { get; set; }
        public Int32 IDLE_TIME { get; set; }
        public string MIDDLE_INITIAL { get; set; }
        public string PHONE1 { get; set; }
        public string FAX { get; set; }
        public string LDAP_USER { get; set; }
        public string USERDN { get; set; }
        public string PHONE2 { get; set; }
        public string PAGER { get; set; }
        public bool REPORT_USER { get; set; }
        public string LDAP_ROLE { get; set; }
        public string LDAP_ORG { get; set; }
        public string PASSWD_RESET_REQUIRED { get; set; }
    }
}
